test/Microsoft.AspNet.Diagnostics.Tests/DeveloperExceptionPageMiddlewareTest.cs

```diff
@@ -35,7 +35,7 @@ public static TheoryData RelativePathsData
                     "TestFiles/SourceFile.txt"
                 };
 
-                if (!TestPlatformHelper.IsLinux)
+                if (!(TestPlatformHelper.IsLinux || TestPlatformHelper.IsMac))
```

Comment:
Yeah I contemplated doing that but I recall someone mentioning that this is not preferred in the scenario where there would be support for more platforms in future and we would know that certain tests fail on them.